server/server.py

```diff
@@ -3,32 +3,74 @@
 import tornado.websocket
 import tornado.template
 
+# WS_HANDLERS maintains a list of currently-opened ws connections
+WS_HANDLERS = []
+
+# maintain message history for new connections
+MESSAGE_HISTORY = []
+
 class MainHandler(tornado.web.RequestHandler):
-  def get(self):
-    loader = tornado.template.Loader(".")
-    self.write(loader.load("../client/index.html").generate())
+    def get(self):
+        loader = tornado.template.Loader(".")
+        self.write(loader.load("../client/index.html").generate())
```

Comment:
I know this is _one_ correct way of doing it.  I followed a simple setup guide for a python websocket server that was serving the index page this way.  I would like to read some more of the tornado documentation to see more ways our page can be served.

If we're still wanting to do a single-page application (I would like that, it makes handling websocket connections _much_ easier), this approach should work once we figure out the css issue